sor/src/test/java/com/bazaarvoice/emodb/sor/core/RedundantDeltaTest.java

```diff
@@ -310,39 +311,39 @@ public void testPartialCompactionWithNoRedundancy() throws Exception {
         dataDao.setFullConsistencyTimestamp(1408977300000L);
 
         // Create an update where there are no redundant deltas
-        UUID unique0 = TimeUUIDs.uuidForTimeMillis(1408977310000L);
-        UUID unique1 = TimeUUIDs.uuidForTimeMillis(1408977320000L);
-        UUID unique2 = TimeUUIDs.uuidForTimeMillis(1408977330000L);
-        UUID unique3 = TimeUUIDs.uuidForTimeMillis(1408977340000L);
+        DeltaClusteringKey unique0 = new DeltaClusteringKey(TimeUUIDs.uuidForTimeMillis(1408977310000L), 1);
+        DeltaClusteringKey unique1 = new DeltaClusteringKey(TimeUUIDs.uuidForTimeMillis(1408977320000L), 1);
+        DeltaClusteringKey unique2 = new DeltaClusteringKey(TimeUUIDs.uuidForTimeMillis(1408977330000L), 1);
+        DeltaClusteringKey unique3 = new DeltaClusteringKey(TimeUUIDs.uuidForTimeMillis(1408977340000L), 1);
 
-        store.update(TABLE, KEY, unique0, Deltas.fromString("{\"name\":\"Bob\"}"), newAudit("submit"), WriteConsistency.STRONG);
-        store.update(TABLE, KEY, unique1, Deltas.fromString("{\"name\":\"Carol\"}"), newAudit("resubmit"), WriteConsistency.STRONG);
-        store.update(TABLE, KEY, unique2, Deltas.fromString("{\"name\":\"Ted\"}"), newAudit("resubmit"), WriteConsistency.STRONG);
-        store.update(TABLE, KEY, unique3, Deltas.fromString("{\"name\":\"Alice\"}"), newAudit("resubmit"), WriteConsistency.STRONG);
+        store.update(TABLE, KEY, unique0.getChangeId(), Deltas.fromString("{\"name\":\"Bob\"}"), newAudit("submit"), WriteConsistency.STRONG);
+        store.update(TABLE, KEY, unique1.getChangeId(), Deltas.fromString("{\"name\":\"Carol\"}"), newAudit("resubmit"), WriteConsistency.STRONG);
+        store.update(TABLE, KEY, unique2.getChangeId(), Deltas.fromString("{\"name\":\"Ted\"}"), newAudit("resubmit"), WriteConsistency.STRONG);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "{\"name\":\"Ted\"}" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS1192)